src/Api/Auth/Models/Request/AdminAuthRequestUpdateRequestModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Bit.Api.Auth.Models.Request;
+
+public class AdminAuthRequestUpdateRequestModel
+{
+    public string EncryptedUserKey { get; set; }
```

Comment:
```suggestion
    [EncryptedString]
    public string EncryptedUserKey { get; set; }
```

Reply:
Oh, didn't know about that attribute, thanks!